plugins/kubernetes/test/models/kubernetes/template_filler_test.rb

```diff
@@ -850,10 +850,22 @@ def secret_annotations(hash)
           )
         end
 
-        it "bumps termination grace period if we would sleep longer that termination allows" do
-          with_env(KUBERNETES_PRESTOP_SLEEP_DURATION: "50") do
+        describe "when pod would terminate before finishing to sleep" do
+          with_env(KUBERNETES_PRESTOP_SLEEP_DURATION: "50")
+
+          it "bumps termination grace period" do
```

Comment:
test doesn't overwrite existing preStop hook

Reply:
`it "overrides termination grace period when user configured too low value" do`

Reply:
we got `it "does not add preStop when it was already defined" do` for that